awsconfigrules/index_ec2_create_trigger_db_update_proxy.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+/**********************************************************************************************
+* This scripts is a proxy lamdba that triggers 'awsconfigrules-ec2-create-trigger-db-update'
+*     lambda in the same region in which the database recides.
+**********************************************************************************************/
+const aws = require('aws-sdk');
+const lambda = new aws.Lambda({
+    region: 'us-east-1'
+});
+
+
+// This is the handler that's invoked by Lambda
+// Most of this code is boilerplate; use as is
+exports.handler = (event, context, callback) => {
+    var params = {
+        ClientContext: "ProxyLambda",
+        FunctionName: "awsconfigrules-ec2-create-trigger-db-update",
+        InvocationType: "Event",
```

Comment:
This is an async invocation, how do we handle if master lambda fails to add/insert the data in DB?